tests/fuzzer.c

```diff
@@ -1978,7 +1978,7 @@ static int basicUnitTests(U32 const seed, double compressibility)
             int const rowLevelEnd = 8;
 
             DISPLAYLEVEL(3, "test%3i : flat-dictionary efficiency test : \n", testNb++);
-            assert(maxLevel == 22);
+            assert(maxLevel == (sizeof(void *) == 8 ? 22 : 21));
```

Comment:
I'm somewhat embarrassed that the logic of max compression level differentiated by system width must be duplicated here. Though on this topic, I don't have a clearly better solution to propose.

Rather, I'm wondering if this `assert()` is actually valid. What if we want to add a level `23` in the future ? Why should it fail this `assert()` ? I don't see how this `assert()` pre-condition makes the rest of the test more "valid".